_client/client_app/angular_app/elements/iwish-dialog/iwish-dialog.html

```diff
@@ -0,0 +1,43 @@
+<link rel="import" href="../../bower_components/paper-dialog/paper-action-dialog.html">
+<link rel="import" href="../../bower_components/paper-radio-group/paper-radio-group.html">
+<link rel="import" href="../../bower_components/core-item/core-item.html">
+
+<polymer-element name="iwish-dialog" atributes="">
+	<template>
+		<section on-tap="{{toggleDialog}}">
```

Comment:
better move(or bind) this to external attribute, so we would be able to toggle it externally from parent